build_config.json

```diff
@@ -1,7 +1,7 @@
 {
   "app_name" : "mux",
   "default_roku_user" : "rokudev",
-  "default_roku_target" : "192.168.5.119",
```

Comment:
It appears this is used in development to connect to a roku that's on a local network?

Reply:
That is correct. I committed too many things 🤦‍♂